tests/analysis/java/test_analysis.py

```diff
@@ -12,7 +12,7 @@
 
 
 @pytest.mark.parametrize("application_data", ci_data(),
-                         ids=lambda case: f"{case['name']}: {case['description']}")
+                         ids=lambda case: f"{case['name']}")
 def test_bvp(application_data):
```

Comment:
Test name is still bvp ? I think we discussed it to be test_analysis . This test has nothing to do with bvp.